web/modules/custom/brms_migrate/src/Plugin/migrate/source/Asset.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\brms_migrate\Plugin\migrate\source;
+
+use Drupal\migrate\Plugin\migrate\source\SqlBase;
+
+/**
+ * Asset source from database.
+ *
+ * @MigrateSource(
+ *   id = "farmos_asset",
```

Comment:
Is this coming from farmos migration?

Reply:
No, it's defined here in brms_migrate

Reply:
Yes, but did we get it from somewhere else or it is developed ad-hoc

Reply:
It's ad-hoc, taking inspiration from Migrate Plus examples